src/main/java/org/reactome/qa/doisuggester/ReactionlikeEvent.java

```diff
@@ -0,0 +1,354 @@
+package org.reactome.qa.doisuggester;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+
+/**
+ * Represents a Reactionlike Event, in the context of DOI suggestion activities.
+ * @author sshorser
+ *
+ */
+public class ReactionlikeEvent
+{
+	// The underlying RLE, in the current database.
+	private GKInstance rle;
+	// Corresponding RLE from the previous Release's database.
+	private ReactionlikeEvent prevRle = null;
+
+	private List<GKInstance> authoredInstancedEdits;
+	private List<GKInstance> reviewedInstancedEdits;
+	private List<GKInstance> revisedInstancedEdits;
+
+	private Set<GKInstance> ancestors = new HashSet<>();
+
+	/**
+	 * Constructor to create a DOISuggester-contexted Reaction-like Event object.
+	 * @param instance - The underlying instance from the database. This is a ReactionlikeEvent object.
+	 * @param prevAdaptor - A database adaptor that should point to the *previous* version of the database. This will be used to look at
+	 * previous versions of the <code>instance</code>
+	 * @throws Exception
+	 */
+	public ReactionlikeEvent(GKInstance instance, MySQLAdaptor prevAdaptor) throws Exception
+	{
+		this.rle = instance;
+		GKInstance prevInstance = prevAdaptor.fetchInstance(this.rle.getDBID());
+		if (prevInstance != null)
+		{
+			ReactionlikeEvent prevDOISuggesterRLE = new ReactionlikeEvent(prevInstance);
+			this.prevRle = prevDOISuggesterRLE;
+		}
+	}
+
+	/**
+	 * Returns true if the underlying RLE instance is new - there is no corresponding instance in the previous Releases's database.
+	 * @return
+	 */
+	public boolean isNewRLE()
+	{
+		return this.prevRle == null;
+	}
+
+	/**
+	 * A constructor that should be used to create an instance of this class that represents a version of a ReactionlikeEvent object
+	 * from a previous database, or a parent of a ReactionlikeEvent from the current database, because it does not take a MySQLAdaptor
+	 * for a previous database as an argument.
+	 * @param instance - The underlying instance from the database.
+	 * @throws Exception
+	 */
+	public ReactionlikeEvent(GKInstance instance) throws Exception
+	{
+		this.rle = instance;
+	}
+
+	/**
+	 * Gets a list of "authored" InstanceEdits for this RLE
+	 * @return
+	 */
+	public List<GKInstance> getAuthoredInstanceEdits()
+	{
+		if (this.authoredInstancedEdits == null)
+		{
+			try
+			{
+				this.authoredInstancedEdits = this.rle.getAttributeValuesList(ReactomeJavaConstants.authored);
+			}
+			catch (Exception e)
+			{
+				e.printStackTrace();
+			}
+		}
+		return this.authoredInstancedEdits;
+	}
+
+	/**
+	 * Gets a list of "reviewed" InstanceEdits for this RLE
+	 * @return
```

Comment:
Same as here: https://github.com/reactome/doi-suggester/pull/4/files#r632670873

Reply:
I think that if the comment for an @return in the javadoc is almost identical to the method's description, it's redundant.